tools/ab_test.py

```diff
@@ -50,6 +50,12 @@
         "host_kernel": "linux-5.10",
         "vcpus": "1",
     },
+    # High volatility due to AMI.
+    {
+        "performance_test": "test_restore_latency",
```

Comment:
I'd remove this key to disable all tests on m6i/6.1 since they're all out of wack right now

Reply:
Only snapshot restore demonstrates this behavior consistently enough to warrant the change imo, I would prefer a more cautious approach, I think we can extend it to other tests if we see that they also consistently demonstrate regressions with the AMI.

Reply:
With the occurrence of new errors in others tests I think I'm convinced to expand this to disable alarming for m6i 6.1 generally, I've updated it to do this.